src/docs/cookbook/testing/integration/scrolling.md

```diff
@@ -8,41 +8,42 @@ next:
   path: /docs/cookbook/testing/unit/introduction
 ---
 
-Many apps feature lists of content, from email clients to music apps and beyond.
-In order to verify that lists contain the content we expect using integration
-tests, we need a way to scroll through lists to search for particular items.
+Many apps feature lists of content, from email clients to music apps
+and beyond.
+To verify that lists contain the expected content using integration
+tests, you need a way to scroll through lists to search for particular items.
 
-In order to scroll through lists via integration tests, we can use the methods
+To scroll through lists via integration tests, use the methods
 provided by the
 [`FlutterDriver`]({{site.api}}/flutter/flutter_driver/FlutterDriver-class.html)
 class, which is included in the
 [`flutter_driver`]({{site.api}}/flutter/flutter_driver/flutter_driver-library.html)
 package:
 
-In this recipe, we'll learn how to scroll through a list of items in order to
-verify a specific Widget is being displayed, and discuss the pros on cons of
+In this recipe, learn how to scroll through a list of items to
+verify a specific widget is being displayed, and discuss the pros on cons of
```

Comment:
a -> that a

I don't understand "and discuss..." You'll learn how to discuss? Also... it'd be "pros AND cons", but that might not be the terminology we'd want to use.